src/libraries/System.Security.Cryptography.X509Certificates/tests/TestData.cs

```diff
@@ -1971,8 +1971,8 @@ internal static DSAParameters GetDSA1024Params()
 -----END CERTIFICATE-----";
 
 
-        public const string OtherRsaPkcs1Key = @"
------BEGIN RSA PRIVATE KEY-----
+        public static readonly string OtherRsaPkcs1Key = $@"
+-----{TestSecretHelper.BEGIN_RSA_PRIVATE_KEY}-----
```

Comment:
Would this work as just being wrapped by `TestSecretHelper.NotASecret(`, so it can stay easy to copy/paste values in?

Reply:
do you mean whole key string?
no, the tool recognise private key by `-----BEGIN RSA PRIVATE KEY-----` in the string. 

we can use `// [SuppressMessage ...]` above this string literal to remove the string interpolation